.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence,
-*       @Jolg42 @millsp @aqrln
+*       @Jolg42 @SevInf @jkomyno @druue
```

Comment:
It seems that they don't have write access to the repo

Reply:
<img width="517" alt="image" src="https://github.com/prisma/engines-wrapper/assets/4923335/fa37bc40-4d23-47a3-a931-fa409e663ab5">

<img width="716" alt="image" src="https://github.com/prisma/engines-wrapper/assets/4923335/4e69b7c8-4ab6-4d56-ac52-8d47dda6d7c8">

Reply:
I asked @apolanc https://prisma-company.slack.com/archives/C058VM009HT/p1709821212095269

Reply:
> Make sure @SevInf exists

Thanks for existential crisis, Github!